.github/workflows/arm-auto-signoff-preview.yaml

```diff
@@ -0,0 +1,60 @@
+name: ARM Auto Signoff (Preview)
+
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - opened
+      - reopened
+      - synchronize
+  workflow_dispatch:
+    inputs:
+      owner:
+        description: The account owner of the repository. The name is not case sensitive.
+        required: true
+        type: string
+      repo:
+        description: The name of the repository without the .git extension. The name is not case sensitive.
+        required: true
+        type: string
+      issue_number:
+        description: The number of the pull request.
+        required: true
+        type: string
+      head_sha:
+        description: The SHA of the commit.
+        required: true
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  arm-auto-signoff-preview:
+    name: ARM Auto Signoff (Preview)
+
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Required to determine files changed in PR
+          fetch-depth: 2
+          # TODO: Temporary to improve perf while iterating
+          sparse-checkout: |
+            .github
+            eng
+
+      - name: ARM Auto Signoff (Preview)
+        uses: actions/github-script@v7
+        env:
+          OWNER: ${{ inputs.owner }}
+          REPO: ${{ inputs.repo }}
+          ISSUE_NUMBER: ${{ inputs.issue_number }}
+          HEAD_SHA: ${{ inputs.head_sha }}
+        with:
+          script: |
+            const { default: getLabelAction } =
+              await import('${{ github.workspace }}/.github/workflows/src/arm-auto-signoff-preview.js');
+            return await getLabelAction({ github, context, core });
```

Comment:
What is happening with this return value?

Reply:
The return value of a `github-script` script is set to the output value of the step.  The output of this step is currently unused, but will soon be passed as input to the `add-label-artifact` action.